pkg/config/config.go

```diff
@@ -110,7 +110,7 @@ type ContainersConfig struct {
 	HooksDir []string `toml:"hooks_dir"`
 
 	// HTTPProxy is the proxy environment variable list to apply to container process
-	HTTPProxy []string `toml:"http_proxy"`
+	HTTPProxy bool `toml:"http_proxy"`
```

Comment:
should also change the HTTPProxy config in the test file  testdata/containers_default.conf

Reply:
Fixed